meta-client/src/main/java/org/apache/helix/metaclient/recipes/leaderelection/LeaderElectionClient.java

```diff
@@ -0,0 +1,180 @@
+package org.apache.helix.metaclient.recipes.leaderelection;
```

Comment:
Since this is a descriptive class for APIs. It can be either interface or abstract class. To my understand, abstract class should be better. wdyt?

Reply:
TFTR. We had an offline discussion. This class will be a solid class to use directly. Will add impl in following change.